$NetBSD: patch-tests_Makefile.am,v 1.1 2018/11/01 09:36:47 bsiegert Exp $

Avoid possible `rm -rf' of the entire file system when running `make check':

 <https://github.com/AbiWord/enchant/pull/203>

--- tests/Makefile.am.orig	2018-02-04 17:01:08.000000000 +0000
+++ tests/Makefile.am
@@ -1,11 +1,7 @@
 SUBDIRS = enchant_providers
 
 # Get libdir suffix
-if GNU_MAKE
-libdir_subdir=$(shell echo "$(libdir)" | sed -e 's|^$(exec_prefix)/||' | sed -e 's|^/||')
-else
 libdir_subdir=lib
-endif
 
 AM_CPPFLAGS = -I$(top_srcdir)/src $(ENCHANT_CFLAGS) -DENCHANT_MAJOR_VERSION=\"@ENCHANT_MAJOR_VERSION@\"
 
@@ -33,7 +29,7 @@ $(LIBENCHANT_COPY): $(top_builddir)/src/
 	$(MAKE) libenchant-copy
 
 libenchant-copy:
-	rm -rf $(libdir_subdir)/
+	rm -rf $(libdir_subdir)
 	$(MKDIR_P) $(libdir_subdir)/enchant-@ENCHANT_MAJOR_VERSION@
 	cp -r $(top_builddir)/src/@objdir@ $(libdir_subdir)/
 	cp $(top_builddir)/src/libenchant-@ENCHANT_MAJOR_VERSION@.la $(libdir_subdir)/
